bleu/op-bacchus
# Jean/op 231 integrate emit creat event attestion when submitting new

How to test:
- Go to /events/new
- Fill the overview form
- Fill the description form
- Accept the transaction
- Go to /events/
- check the new attestation is there
